hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/filtering/FilteringException.java

```diff
@@ -0,0 +1,21 @@
+package pl.allegro.tech.hermes.consumers.consumer.filtering;
+
+import pl.allegro.tech.hermes.common.exception.InternalProcessingException;
+
+public class FilteringException extends InternalProcessingException {
+    public FilteringException(Throwable throwable) {
+        super(throwable);
+    }
+
+    public FilteringException(String message) {
+        super(message);
+    }
+
+    public FilteringException(String message, Throwable throwable) {
+        super(message, throwable);
+    }
+
+    public static void assertTrue(boolean condition, String message) {
```

Comment:
Would you consider calling this `throwIfNot`? I got confused what would happen when I saw this assert used with static import.